.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Thank you for making Rust GCC better!
+
+If your PR fixes an issue, you can add "Fixes #issue_number" into this
+PR description and the git commit message. This way the issue will be
+automatically closed when your PR is merged. If your change addresses
+an issue but does not fully fix it please mark it as "Addresses #issue_number"
+in the git commit message.
+
+Here is a checklist to help you with your PR.
+
+- \[ ] Read contributing guidlines
```

Comment:
Why the backslashes?

Reply:
I am not sure but its copied from https://raw.githubusercontent.com/rust-lang/rust-clippy/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
https://github.com/rust-lang/rust-clippy/pull/6175

Seems that remark complained about those. Not sure why, but this was the easiest way to fix our CI.